cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -235,6 +275,8 @@ spec:
     metadata:
       labels:
         k8s-app: kube-dns
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
```

Comment:
could you provide details on this annotation?

Reply:
All of the containers running in the pod will specified with this seccomp profile. From what I understand, the applications running inside the container on which system calls it can make.

Reply:
given there was no demand for this annotation, i suggest that we remove it.